pkg/vbmh/machines.go

```diff
@@ -97,15 +97,21 @@ func (m *Machine) String() string {
 	return fmt.Sprintf("Machine {\n\tBmh:%s\n\tScheduleStatus:%s\n\tVmRole:%v\n}\n", m.BMH.ObjectMeta.Name, m.ScheduleStatus, m.VMRole)
 }
 
-func NewMachine(bmh metal3.BareMetalHost, nodeRole airshipv1.VmRoles, schedState ScheduledState) (m *Machine) {
+func NewMachine(bmh metal3.BareMetalHost, nodeRole airshipv1.VmRoles, schedState ScheduledState) (m *Machine, e *ErrorNetworkDataNotFound) {
```

Comment:
i think return signature should be (m *Machine, e error) instead

Reply:
done